src/slices/themeSlice.ts

```diff
@@ -0,0 +1,16 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const initialTheme = {
+    theme: 'light',
+};
+
+export const themeSlice = createSlice({
+    name: 'theme',
+    initialState: initialTheme,
+    reducers: {
+        toggleTheme: (state) => state.theme === 'light' ? {...state, theme: 'dark'} : {...state,theme:'light'}
```

Comment:
write the function outside so it will look cleaner

Reply:
donr